package data.memberdata;

import java.util.ArrayList;
import dataenum.ResultMessage;
import org.junit.*;
import dataenum.MemberCategory;
import dataenum.MemberLevel;
import dataenum.findtype.FindMemberType;
import po.memberpo.MemberPO;

import static org.junit.Assert.*;

/**
 * The class <code>MemberDataTest</code> contains tests for the class
 * <code>{@link MemberData}</code>.
 *
 * @generatedBy CodePro at 18-1-12 下午9:28
 * @author asus
 * @version $Revision: 1.0 $
 */
public class MemberDataTest {
	/**
	 * Run the ResultMessage delete(String) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testDelete_1() throws Exception {
		MemberData fixture = new MemberData();
		String id = "";

		ResultMessage result = fixture.delete(id);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.delete(MemberData.java:74)
		assertNotNull(result);
	}

	/**
	 * Run the ResultMessage delete(String) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testDelete_2() throws Exception {
		MemberData fixture = new MemberData();
		String id = "";

		ResultMessage result = fixture.delete(id);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.delete(MemberData.java:74)
		assertNotNull(result);
	}

	/**
	 * Run the ResultMessage delete(String) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testDelete_3() throws Exception {
		MemberData fixture = new MemberData();
		String id = "";

		ResultMessage result = fixture.delete(id);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.delete(MemberData.java:74)
		assertNotNull(result);
	}

	/**
	 * Run the ResultMessage delete(String) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testDelete_4() throws Exception {
		MemberData fixture = new MemberData();
		String id = "";

		ResultMessage result = fixture.delete(id);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.delete(MemberData.java:74)
		assertNotNull(result);
	}

	/**
	 * Run the ResultMessage delete(String) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testDelete_5() throws Exception {
		MemberData fixture = new MemberData();
		String id = "";

		ResultMessage result = fixture.delete(id);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.delete(MemberData.java:74)
		assertNotNull(result);
	}

	/**
	 * Run the ResultMessage delete(String) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testDelete_6() throws Exception {
		MemberData fixture = new MemberData();
		String id = "";

		ResultMessage result = fixture.delete(id);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.delete(MemberData.java:74)
		assertNotNull(result);
	}

	/**
	 * Run the ArrayList<MemberPO> find(String,FindMemberType) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testFind_1() throws Exception {
		MemberData fixture = new MemberData();
		String keyword = "";
		FindMemberType type = FindMemberType.ADDRESS;

		ArrayList<MemberPO> result = fixture.find(keyword, type);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.find(MemberData.java:108)
		assertNotNull(result);
	}

	/**
	 * Run the ArrayList<MemberPO> find(String,FindMemberType) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testFind_2() throws Exception {
		MemberData fixture = new MemberData();
		String keyword = "";
		FindMemberType type = FindMemberType.ADDRESS;

		ArrayList<MemberPO> result = fixture.find(keyword, type);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.find(MemberData.java:108)
		assertNotNull(result);
	}

	/**
	 * Run the ArrayList<MemberPO> find(String,FindMemberType) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testFind_3() throws Exception {
		MemberData fixture = new MemberData();
		String keyword = "";
		FindMemberType type = FindMemberType.ADDRESS;

		ArrayList<MemberPO> result = fixture.find(keyword, type);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.find(MemberData.java:108)
		assertNotNull(result);
	}

	/**
	 * Run the ArrayList<MemberPO> find(String,FindMemberType) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testFind_4() throws Exception {
		MemberData fixture = new MemberData();
		String keyword = "";
		FindMemberType type = FindMemberType.ADDRESS;

		ArrayList<MemberPO> result = fixture.find(keyword, type);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.find(MemberData.java:108)
		assertNotNull(result);
	}

	/**
	 * Run the ArrayList<MemberPO> find(String,FindMemberType) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testFind_5() throws Exception {
		MemberData fixture = new MemberData();
		String keyword = "";
		FindMemberType type = FindMemberType.ADDRESS;

		ArrayList<MemberPO> result = fixture.find(keyword, type);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.find(MemberData.java:108)
		assertNotNull(result);
	}

	/**
	 * Run the ArrayList<MemberPO> find(String,FindMemberType) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testFind_6() throws Exception {
		MemberData fixture = new MemberData();
		String keyword = "";
		FindMemberType type = FindMemberType.ADDRESS;

		ArrayList<MemberPO> result = fixture.find(keyword, type);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.find(MemberData.java:108)
		assertNotNull(result);
	}

	/**
	 * Run the ArrayList<MemberPO> find(String,FindMemberType) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testFind_7() throws Exception {
		MemberData fixture = new MemberData();
		String keyword = "";
		FindMemberType type = FindMemberType.ADDRESS;

		ArrayList<MemberPO> result = fixture.find(keyword, type);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.find(MemberData.java:108)
		assertNotNull(result);
	}

	/**
	 * Run the ArrayList<MemberPO> find(String,FindMemberType) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testFind_8() throws Exception {
		MemberData fixture = new MemberData();
		String keyword = "";
		FindMemberType type = FindMemberType.ADDRESS;

		ArrayList<MemberPO> result = fixture.find(keyword, type);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.find(MemberData.java:108)
		assertNotNull(result);
	}

	/**
	 * Run the ArrayList<MemberPO> find(String,FindMemberType) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testFind_9() throws Exception {
		MemberData fixture = new MemberData();
		String keyword = "";
		FindMemberType type = FindMemberType.ADDRESS;

		ArrayList<MemberPO> result = fixture.find(keyword, type);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.find(MemberData.java:108)
		assertNotNull(result);
	}

	/**
	 * Run the ArrayList<MemberPO> find(String,FindMemberType) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testFind_10() throws Exception {
		MemberData fixture = new MemberData();
		String keyword = "";
		FindMemberType type = FindMemberType.ADDRESS;

		ArrayList<MemberPO> result = fixture.find(keyword, type);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.find(MemberData.java:108)
		assertNotNull(result);
	}

	/**
	 * Run the ArrayList<MemberPO> find(String,FindMemberType) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testFind_11() throws Exception {
		MemberData fixture = new MemberData();
		String keyword = "";
		FindMemberType type = FindMemberType.ADDRESS;

		ArrayList<MemberPO> result = fixture.find(keyword, type);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.find(MemberData.java:108)
		assertNotNull(result);
	}

	/**
	 * Run the ArrayList<MemberPO> find(String,FindMemberType) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testFind_12() throws Exception {
		MemberData fixture = new MemberData();
		String keyword = "";
		FindMemberType type = FindMemberType.ADDRESS;

		ArrayList<MemberPO> result = fixture.find(keyword, type);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.find(MemberData.java:108)
		assertNotNull(result);
	}

	/**
	 * Run the ArrayList<MemberPO> find(String,FindMemberType) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testFind_13() throws Exception {
		MemberData fixture = new MemberData();
		String keyword = "";
		FindMemberType type = FindMemberType.ADDRESS;

		ArrayList<MemberPO> result = fixture.find(keyword, type);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.find(MemberData.java:108)
		assertNotNull(result);
	}

	/**
	 * Run the ArrayList<MemberPO> find(String,FindMemberType) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testFind_14() throws Exception {
		MemberData fixture = new MemberData();
		String keyword = "";
		FindMemberType type = FindMemberType.ADDRESS;

		ArrayList<MemberPO> result = fixture.find(keyword, type);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.find(MemberData.java:108)
		assertNotNull(result);
	}

	/**
	 * Run the ArrayList<MemberPO> find(String,FindMemberType) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testFind_15() throws Exception {
		MemberData fixture = new MemberData();
		String keyword = "";
		FindMemberType type = FindMemberType.ADDRESS;

		ArrayList<MemberPO> result = fixture.find(keyword, type);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.find(MemberData.java:108)
		assertNotNull(result);
	}

	/**
	 * Run the ArrayList<MemberPO> find(String,FindMemberType) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testFind_16() throws Exception {
		MemberData fixture = new MemberData();
		String keyword = "";
		FindMemberType type = FindMemberType.ADDRESS;

		ArrayList<MemberPO> result = fixture.find(keyword, type);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.find(MemberData.java:108)
		assertNotNull(result);
	}

	/**
	 * Run the ResultMessage insert(MemberPO) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testInsert_1() throws Exception {
		MemberData fixture = new MemberData();
		MemberPO po = new MemberPO("", MemberCategory.BOTH, MemberLevel.LEVEL1, "", "", "", "", "", 1.0, "");

		ResultMessage result = fixture.insert(po);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.insert(MemberData.java:40)
		assertNotNull(result);
	}

	/**
	 * Run the ResultMessage insert(MemberPO) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testInsert_2() throws Exception {
		MemberData fixture = new MemberData();
		MemberPO po = new MemberPO("", MemberCategory.BOTH, MemberLevel.LEVEL1, "", "", "", "", "", 1.0, "");

		ResultMessage result = fixture.insert(po);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.insert(MemberData.java:40)
		assertNotNull(result);
	}

	/**
	 * Run the ResultMessage insert(MemberPO) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testInsert_3() throws Exception {
		MemberData fixture = new MemberData();
		MemberPO po = new MemberPO("", MemberCategory.BOTH, MemberLevel.LEVEL1, "", "", "", "", "", 1.0, "");

		ResultMessage result = fixture.insert(po);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.insert(MemberData.java:40)
		assertNotNull(result);
	}

	/**
	 * Run the ResultMessage insert(MemberPO) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testInsert_4() throws Exception {
		MemberData fixture = new MemberData();
		MemberPO po = new MemberPO("", MemberCategory.BOTH, MemberLevel.LEVEL1, "", "", "", "", "", 1.0, "");

		ResultMessage result = fixture.insert(po);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.insert(MemberData.java:40)
		assertNotNull(result);
	}

	/**
	 * Run the ResultMessage insert(MemberPO) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testInsert_5() throws Exception {
		MemberData fixture = new MemberData();
		MemberPO po = new MemberPO("", MemberCategory.BOTH, MemberLevel.LEVEL1, "", "", "", "", "", 1.0, "");

		ResultMessage result = fixture.insert(po);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.insert(MemberData.java:40)
		assertNotNull(result);
	}

	/**
	 * Run the ResultMessage insert(MemberPO) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testInsert_6() throws Exception {
		MemberData fixture = new MemberData();
		MemberPO po = new MemberPO("", MemberCategory.BOTH, MemberLevel.LEVEL1, "", "", "", "", "", 1.0, "");

		ResultMessage result = fixture.insert(po);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.insert(MemberData.java:40)
		assertNotNull(result);
	}

	/**
	 * Run the ResultMessage insert(MemberPO) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testInsert_7() throws Exception {
		MemberData fixture = new MemberData();
		MemberPO po = new MemberPO("", MemberCategory.BOTH, MemberLevel.LEVEL1, "", "", "", "", "", 1.0, "");

		ResultMessage result = fixture.insert(po);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.insert(MemberData.java:40)
		assertNotNull(result);
	}

	/**
	 * Run the ResultMessage insert(MemberPO) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testInsert_8() throws Exception {
		MemberData fixture = new MemberData();
		MemberPO po = new MemberPO("", MemberCategory.BOTH, MemberLevel.LEVEL1, "", "", "", "", "", 1.0, "");

		ResultMessage result = fixture.insert(po);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.insert(MemberData.java:40)
		assertNotNull(result);
	}

	/**
	 * Run the ResultMessage insert(MemberPO) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testInsert_9() throws Exception {
		MemberData fixture = new MemberData();
		MemberPO po = new MemberPO("", MemberCategory.BOTH, MemberLevel.LEVEL1, "", "", "", "", "", 1.0, "");

		ResultMessage result = fixture.insert(po);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.insert(MemberData.java:40)
		assertNotNull(result);
	}

	/**
	 * Run the ResultMessage insert(MemberPO) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testInsert_10() throws Exception {
		MemberData fixture = new MemberData();
		MemberPO po = new MemberPO("", MemberCategory.BOTH, MemberLevel.LEVEL1, "", "", "", "", "", 1.0, "");

		ResultMessage result = fixture.insert(po);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.insert(MemberData.java:40)
		assertNotNull(result);
	}

	/**
	 * Run the ResultMessage insert(MemberPO) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testInsert_11() throws Exception {
		MemberData fixture = new MemberData();
		MemberPO po = new MemberPO("", MemberCategory.BOTH, MemberLevel.LEVEL1, "", "", "", "", "", 1.0, "");

		ResultMessage result = fixture.insert(po);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.insert(MemberData.java:40)
		assertNotNull(result);
	}

	/**
	 * Run the ResultMessage insert(MemberPO) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testInsert_12() throws Exception {
		MemberData fixture = new MemberData();
		MemberPO po = new MemberPO("", MemberCategory.BOTH, MemberLevel.LEVEL1, "", "", "", "", "", 1.0, "");

		ResultMessage result = fixture.insert(po);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.insert(MemberData.java:40)
		assertNotNull(result);
	}

	/**
	 * Run the ResultMessage insert(MemberPO) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testInsert_13() throws Exception {
		MemberData fixture = new MemberData();
		MemberPO po = new MemberPO("", MemberCategory.BOTH, MemberLevel.LEVEL1, "", "", "", "", "", 1.0, "");

		ResultMessage result = fixture.insert(po);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.insert(MemberData.java:40)
		assertNotNull(result);
	}

	/**
	 * Run the ResultMessage insert(MemberPO) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testInsert_14() throws Exception {
		MemberData fixture = new MemberData();
		MemberPO po = new MemberPO("", MemberCategory.BOTH, MemberLevel.LEVEL1, "", "", "", "", "", 1.0, "");

		ResultMessage result = fixture.insert(po);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.insert(MemberData.java:40)
		assertNotNull(result);
	}

	/**
	 * Run the ResultMessage insert(MemberPO) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testInsert_15() throws Exception {
		MemberData fixture = new MemberData();
		MemberPO po = new MemberPO("", MemberCategory.BOTH, MemberLevel.LEVEL1, "", "", "", "", "", 1.0, "");

		ResultMessage result = fixture.insert(po);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.insert(MemberData.java:40)
		assertNotNull(result);
	}

	/**
	 * Run the void main(String[]) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testMain_1() throws Exception {
		String[] args = new String[] {};

		MemberData.main(args);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.insert(MemberData.java:40)
		// at data.memberdata.MemberData.main(MemberData.java:34)
	}

	/**
	 * Run the ArrayList<MemberPO> show() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testShow_1() throws Exception {
		MemberData fixture = new MemberData();

		ArrayList<MemberPO> result = fixture.show();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.show(MemberData.java:198)
		assertNotNull(result);
	}

	/**
	 * Run the ArrayList<MemberPO> show() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testShow_2() throws Exception {
		MemberData fixture = new MemberData();

		ArrayList<MemberPO> result = fixture.show();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.show(MemberData.java:198)
		assertNotNull(result);
	}

	/**
	 * Run the ArrayList<MemberPO> show() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testShow_3() throws Exception {
		MemberData fixture = new MemberData();

		ArrayList<MemberPO> result = fixture.show();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.show(MemberData.java:198)
		assertNotNull(result);
	}

	/**
	 * Run the ArrayList<MemberPO> show() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testShow_4() throws Exception {
		MemberData fixture = new MemberData();

		ArrayList<MemberPO> result = fixture.show();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.show(MemberData.java:198)
		assertNotNull(result);
	}

	/**
	 * Run the ArrayList<MemberPO> show() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testShow_5() throws Exception {
		MemberData fixture = new MemberData();

		ArrayList<MemberPO> result = fixture.show();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.show(MemberData.java:198)
		assertNotNull(result);
	}

	/**
	 * Run the ArrayList<MemberPO> show() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testShow_6() throws Exception {
		MemberData fixture = new MemberData();

		ArrayList<MemberPO> result = fixture.show();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.show(MemberData.java:198)
		assertNotNull(result);
	}

	/**
	 * Run the ArrayList<MemberPO> show() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testShow_7() throws Exception {
		MemberData fixture = new MemberData();

		ArrayList<MemberPO> result = fixture.show();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.show(MemberData.java:198)
		assertNotNull(result);
	}

	/**
	 * Run the ArrayList<MemberPO> show() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testShow_8() throws Exception {
		MemberData fixture = new MemberData();

		ArrayList<MemberPO> result = fixture.show();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.show(MemberData.java:198)
		assertNotNull(result);
	}

	/**
	 * Run the ArrayList<MemberPO> show() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testShow_9() throws Exception {
		MemberData fixture = new MemberData();

		ArrayList<MemberPO> result = fixture.show();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.show(MemberData.java:198)
		assertNotNull(result);
	}

	/**
	 * Run the ArrayList<MemberPO> show() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testShow_10() throws Exception {
		MemberData fixture = new MemberData();

		ArrayList<MemberPO> result = fixture.show();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.show(MemberData.java:198)
		assertNotNull(result);
	}

	/**
	 * Run the ArrayList<MemberPO> show() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testShow_11() throws Exception {
		MemberData fixture = new MemberData();

		ArrayList<MemberPO> result = fixture.show();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.show(MemberData.java:198)
		assertNotNull(result);
	}

	/**
	 * Run the ArrayList<MemberPO> show() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testShow_12() throws Exception {
		MemberData fixture = new MemberData();

		ArrayList<MemberPO> result = fixture.show();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.show(MemberData.java:198)
		assertNotNull(result);
	}

	/**
	 * Run the ArrayList<MemberPO> show() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testShow_13() throws Exception {
		MemberData fixture = new MemberData();

		ArrayList<MemberPO> result = fixture.show();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.show(MemberData.java:198)
		assertNotNull(result);
	}

	/**
	 * Run the ArrayList<MemberPO> show() method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testShow_14() throws Exception {
		MemberData fixture = new MemberData();

		ArrayList<MemberPO> result = fixture.show();

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.show(MemberData.java:198)
		assertNotNull(result);
	}

	/**
	 * Run the ResultMessage update(MemberPO) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testUpdate_1() throws Exception {
		MemberData fixture = new MemberData();
		MemberPO po = new MemberPO("", MemberCategory.BOTH, MemberLevel.LEVEL1, "", "", "", "", "", 1.0, "");

		ResultMessage result = fixture.update(po);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.update(MemberData.java:90)
		assertNotNull(result);
	}

	/**
	 * Run the ResultMessage update(MemberPO) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testUpdate_2() throws Exception {
		MemberData fixture = new MemberData();
		MemberPO po = new MemberPO("", MemberCategory.BOTH, MemberLevel.LEVEL1, "", "", "", "", "", 1.0, "");

		ResultMessage result = fixture.update(po);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.update(MemberData.java:90)
		assertNotNull(result);
	}

	/**
	 * Run the ResultMessage update(MemberPO) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testUpdate_3() throws Exception {
		MemberData fixture = new MemberData();
		MemberPO po = new MemberPO("", MemberCategory.BOTH, MemberLevel.LEVEL1, "", "", "", "", "", 1.0, "");

		ResultMessage result = fixture.update(po);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.update(MemberData.java:90)
		assertNotNull(result);
	}

	/**
	 * Run the ResultMessage update(MemberPO) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testUpdate_4() throws Exception {
		MemberData fixture = new MemberData();
		MemberPO po = new MemberPO("", MemberCategory.BOTH, MemberLevel.LEVEL1, "", "", "", "", "", 1.0, "");

		ResultMessage result = fixture.update(po);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.update(MemberData.java:90)
		assertNotNull(result);
	}

	/**
	 * Run the ResultMessage update(MemberPO) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testUpdate_5() throws Exception {
		MemberData fixture = new MemberData();
		MemberPO po = new MemberPO("", MemberCategory.BOTH, MemberLevel.LEVEL1, "", "", "", "", "", 1.0, "");

		ResultMessage result = fixture.update(po);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.update(MemberData.java:90)
		assertNotNull(result);
	}

	/**
	 * Run the ResultMessage update(MemberPO) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testUpdate_6() throws Exception {
		MemberData fixture = new MemberData();
		MemberPO po = new MemberPO("", MemberCategory.BOTH, MemberLevel.LEVEL1, "", "", "", "", "", 1.0, "");

		ResultMessage result = fixture.update(po);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.update(MemberData.java:90)
		assertNotNull(result);
	}

	/**
	 * Run the ResultMessage update(MemberPO) method test.
	 *
	 * @throws Exception
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Test
	public void testUpdate_7() throws Exception {
		MemberData fixture = new MemberData();
		MemberPO po = new MemberPO("", MemberCategory.BOTH, MemberLevel.LEVEL1, "", "", "", "", "", 1.0, "");

		ResultMessage result = fixture.update(po);

		// add additional test code here
		// An unexpected exception was thrown in user code while executing this
		// test:
		// java.lang.NullPointerException
		// at data.memberdata.MemberData.update(MemberData.java:90)
		assertNotNull(result);
	}

	/**
	 * Perform pre-test initialization.
	 *
	 * @throws Exception
	 *             if the initialization fails for some reason
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@Before
	public void setUp() throws Exception {
		// add additional set up code here
	}

	/**
	 * Perform post-test clean-up.
	 *
	 * @throws Exception
	 *             if the clean-up fails for some reason
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	@After
	public void tearDown() throws Exception {
		// Add additional tear down code here
	}

	/**
	 * Launch the test.
	 *
	 * @param args
	 *            the command line arguments
	 *
	 * @generatedBy CodePro at 18-1-12 下午9:28
	 */
	public static void main(String[] args) {
		new org.junit.runner.JUnitCore().run(MemberDataTest.class);
	}
}